lib/CurrencyExchangeRateFields/CurrencyExchangeRateFields.js

```diff
@@ -31,13 +41,22 @@ const CurrencyExchangeRateFields = ({
   isSetExchangeRateNonIntaractive,
   isTooltipTextExchangeRate,
   isUseExangeRateDisabled,
+  hiddenFields,
+  hiddenFieldsNamePrefix,
 }) => {
   const [isExchangeRateEnabled, setExchangeRateEnabled] = useState();
   const [isExchangeRateRequired, setExchangeRateRequired] = useState();
   const stripes = useStripes();
   const { change } = useForm();
   const { values } = useFormState();
 
+  const visibleFields = useMemo(() => ({
```

Comment:
what is the purpose of this object, especially with memoization?

why not just `<IfFieldVisible visible={! hiddenFields.fieldName}` ?

Reply:
filed names are compound names Ex: `cost.currentExchangeRate`, `cost.useSetExchangeRate`

Reply:
and? you don't need this object